net-p2p/biglybt/biglybt-3.7.0.0.ebuild

```diff
@@ -62,20 +64,19 @@ src_prepare() {
 	# https://github.com/BiglySoftware/BiglyBT/pull/2611
 	rm -r core/src/com/biglybt/core/util/spi || die
 
-	# java-pkg-simple.eclass expects resources in a separate directory.
-	cp -r core/{src,resources} || die
-	rm -r core/resources/META-INF || die "deleting META-INF"
-	find core/resources -type f -name '*.java' -exec rm -rf {} + || die "core deleting *.java"
-	find core/resources -type f -name '.editorconfig' -exec rm -rf {} + || die "deleting .editorconfig"
+	cp -r --reflink=always core/{src,resources} || die
+	rm -r core/resources/META-INF || die
+	find core/resources -type f -name "*.java" -delete || die
+	find core/resources -type f -name '.editorconfig' -delete || die
 
-	cp -r uis/{src,resources} || die
-	find uis/resources -type f -name '*.java' -exec rm -rf {} + || die "uis deleting *.java"
+	cp -r --reflink=always uis/{src,resources} || die
```

Comment:
`--reflink=always` won't work on strange filesystems, but why bother? Modern coreutils already defaults to `--reflink=auto`.